gatsby/src/pages/404.tsx

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import { Layout } from "../components/Layout/Layout";
+import { SEO } from "../components/SEO/SEO";
+
+const NotFoundPage: React.FC = () => {
+	return (
+		<Layout>
+			<meta name="robots" content="noindex" />
```

Comment:
This should be in the head but putting it directly inside Layout will render it in the body. We'll probably need to update the SEO component with a new prop, e.g. `noIndex`, which we use within SEO to then in turn pass a meta tag to Helmet, allowing us to render things in the head.